File: src/main/java/com/gitlab/daring/image/command/drawing/DrawCommands.java
package com.gitlab.daring.image.command.drawing;

import com.gitlab.daring.image.command.CommandRegistry;

public class DrawCommands {

    public static void register(CommandRegistry r) {
        DrawCommands f = new DrawCommands();
        r.register("drawCenter", DrawCenterCommand::new);
    }

}

File: src/main/kotlin/com/gitlab/daring/image/command/drawing/DrawCommands.kt
package com.gitlab.daring.image.command.drawing

import com.gitlab.daring.image.command.CommandRegistry

object DrawCommands {

    fun register(r: CommandRegistry) {
        r.register("drawCenter", ::DrawCenterCommand)
        r.register("drawVar", ::DrawVarCommand)
    }

}